frontend/src/Editor/AppVersionsManager/AppVersionsManager.jsx

```diff
@@ -61,6 +63,7 @@ export const AppVersionsManager = function ({
       .then((data) => {
         const isCurrentVersionReleased = data.currentVersionId ? true : false;
         setAppDefinitionFromVersion(data, isCurrentVersionReleased);
+        setPreviewData('');
```

Comment:
Can we also clear selectedDataSource and dataQuery here similar to preview data?